source/plugins/go/src/oms.go

```diff
@@ -1308,6 +1382,9 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 
 	numContainerLogRecords := 0
 
+	Log("StdoutIncludeSystemResourceSet : %v, StderrIncludeSystemResourceSet: %v, StdoutIncludeSystemNamespaceSet: %v, StderrIncludeSystemNamespaceSet: %v, PodNameToDSDeploymentNameMap: %v",
```

Comment:
We should add telemetry to track what system namespaces and pods customer using this feature.